<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConstraintManagerWidget</class>
 <widget class="QWidget" name="ConstraintManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Constraint Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <property name="spacing">
    <number>10</number>
   </property>
   <item>
    <widget class="QGroupBox" name="selectionGroup">
     <property name="title">
      <string>Selection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="infoLabel">
        <property name="text">
         <string>Select source(s) then target(s)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="selectionLabel">
        <property name="text">
         <string>Selected Objects:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="selectionList">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="refreshButton">
          <property name="text">
           <string>Refresh Selection</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setSourceButton">
          <property name="text">
           <string>Set as Source(s)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="constraintGroup">
     <property name="title">
      <string>Create Constraint</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="typeLabel">
        <property name="text">
         <string>Constraint Type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="constraintTypeCombo">
        <item>
         <property name="text">
          <string>Parent/Child</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Position</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rotation</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Aim</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Relation</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="weightLabel">
        <property name="text">
         <string>Weight:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSlider" name="weightSlider">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="weightValueLabel">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>100%</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="createConstraintButton">
        <property name="text">
         <string>Create Constraint</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="snapButton">
        <property name="text">
         <string>Snap (Active)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="templatesGroup">
     <property name="title">
      <string>Constraint Templates</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="templateNameLabel">
          <property name="text">
           <string>Template Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="templateNameEdit">
          <property name="text">
           <string>MyConstraintSetup</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="saveTemplateButton">
          <property name="text">
           <string>Save Template</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadTemplateButton">
          <property name="text">
           <string>Apply Template</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteTemplateButton">
          <property name="text">
           <string>Delete Template</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
